mu-plugins/central-hub/tests/phpunit/unit/meta-data/getMetaBoxKeys.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Tests for the function get_meta_box_keys().
+ *
+ * @package     spiralWebDb\centralHub\Tests\Unit\Metadata
+ * @since       1.3.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Unit\Metadata;
+
+use Brain\Monkey;
+use function spiralWebDB\Metadata\get_meta_box_keys;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_GetMetaBoxKeys
+ *
+ * @package spiralWebDb\centralHub\Tests\Unit\Metadata
+ * @group   meta-data
+ */
+class Tests_GetMetaBoxKeys extends Test_Case {
+
+	/**
+	 * Prepare the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once CENTRAL_HUB_ROOT_DIR . '/src/meta-data/helpers.php';
+	}
+
+	/**
+	 * Test get_meta_box_keys() returns empty array when store key is empty or does not start with 'meta_box.'
+	 */
+	public function test_returns_empty_array_when_store_key_is_empty_or_does_not_begin_with_meta_box() {
+		$expected = [];
+		get_meta_box_keys();
+
+		$this->assertSame( $expected, get_meta_box_keys() );
+	}
+
+	/**
+	 * Test get_meta_box_keys() returns all array keys that start with `meta_box.`
+	 */
+	public function test_returns_all_array_keys_that_start_with_meta_box() {
+		Monkey\Functions\expect( 'KnowTheCode\ConfigStore\getAllKeys' )
```

Comment:
@rgadon107 The mocks are not correct for this test. Look at the code:

https://github.com/rgadon107/cornerstone/blob/294d568bc42a7fc19242cb0a362bb0f9f5a000fe/mu-plugins/central-hub/src/meta-data/helpers.php#L23-L25

Notice that it only has one dependent function, ie `KnowTheCode\ConfigStore\ getAllKeysStartingWith()`. You want to mock that function.

Reply:
Updated.